README.md

```diff
@@ -1,18 +1,36 @@
 # Mautic custom email settings plugin
-This feature allows Mautic to send different emails through different mail API keys.
+This feature allows Mautic to send different emails through different mail API keys and different email services. Currently plugin support only Sparkpost and Sendgrid API services.
 
 ## Installation via .zip
 1. Download the [master.zip](https://github.com/1FF/mautic-custom-email-settings/archive/master.zip), extract it into the `plugins/` directory and rename the new directory to `CustomEmailSettingsBundle`.
 2. Clear the cache via console command `php app/console cache:clear --env=prod` (might take a while) *OR* manually delete the `app/cache/prod` directory.
 
+## Update
+1. Remove `plugins/CustomEmailSettingsBundle` directory.
+2. Download the [master.zip](https://github.com/1FF/mautic-custom-email-settings/archive/master.zip), extract it into the `plugins/` directory and rename the new directory to `CustomEmailSettingsBundle`.
+3. Clear the cache via console command `php app/console cache:clear --env=prod` (might take a while) *OR* manually delete the `app/cache/prod` directory.
+4. Run `php bin/console mautic:plugins:install` in root Mautic directory.
+
 ## Configuration
-Navigate to Settings. You should see the "Email Api Keys" menu in your settings:
+Choose "Multiple Transport" in Configuration / Email Settings
+
+![image](https://user-images.githubusercontent.com/42058438/193573356-192a4d4b-4484-46a0-9244-6af6e2241ed8.png)
+
+Specify correct `E-mail address to send mail from` and add `API key` which will be used as default (required).
+
+Navigate to Settings Panel. You should see the "Email Api Keys" menu in your settings:
 
 -   ![mautic1](https://user-images.githubusercontent.com/28507711/191930660-b6a1136c-e84a-41e2-b0d3-b2d5f22c9980.png)
 
 In the plugin configuration you can add a different API key for the given mail template or you can leave it blank so that the default API key will be used:
 
--   ![mautic2](https://user-images.githubusercontent.com/28507711/191938721-9169d161-9b96-4c3f-b45c-355bf98105fa.png)
+-   ![2022-10-03_15-02](https://user-images.githubusercontent.com/42058438/193572606-a41a9fa4-82cd-4dc5-9e7d-51e276012a64.png)
+
+Please note:
+* It is important to specify correct `From address` for emails, where custom API key will be using (Channels > Emails > Edit Email > Advanced tab).
+* To delete custom API key, just clear the "key" field and save;
+* If Service not chosen, default will be using;
+* Default Service is Sparkpost, but if you need to use SendGrid API by default, add `'mailer_custom_default_transport' => 'mautic.transport.sendgrid_api',` string to the `app/config/local.php` file
```

Comment:
You can remove this row, because in future we can make functionality to select the default service by the admin panel. We don't want our clients to write code in any config file. For now we can do it but in future we will make it better for usage.